unit-tests/rsutils/time/test-l500-mfr.cpp

```diff
@@ -9,12 +9,13 @@
 
 
 #include "common.h"
-#include <utilities/time/work-week.h>
+#include <rsutils/time/work-week.h>
 #include "../common/utilities/time/l500/get-mfr-ww.h"
 
 #define INVALID_MESSAGE( serial, invalid ) "Invalid serial number \"" + serial + "\" " + invalid
 
-using namespace utilities::time;
+using namespace rsutils::time;    // from rsutils/
+using namespace utilities::time;  // from common/
```

Comment:
We have 2 time classes?

Reply:
Not two classes.
We had one namespace, and the stuff in common/ added to it.
I didn't touch common/, so it stayed in the `utilities` namespace, while the stuff in `rsutils` moved.
`common/` adds device-specific (L500 mainly) logic, which doesn't belong in rsutils.

Reply:
OK.. I must say it's confusing.